File: 0001 Code Chef/0022 Fitness/implementation/0005 kotlin/Fitness.kt
fun main(args:Array<String>){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var test:Int? = stringInput.toIntOrNull();
        if(test != null){
            var count:Int = test;
            while(count > 0){
                var stringInput:String? = readLine();
                if(stringInput!= null){
                    var distance:Int? = stringInput.toIntOrNull();
                     if(distance != null){
                        var distance_travel_per_day:Int = distance * 2;
                        var distance_travel_per_week:Int = distance_travel_per_day * 5;
                        println(distance_travel_per_week);
                    }
                }
                    count = count - 1;
            }
        }
    }
}

File: 0001 Code Chef/0022 Fitness/implementation/0007 ruby/fitness.rb
test = gets.chomp.to_i
while(test > 0) do
    distance = gets.chomp.to_i;
    distance_travel_per_day = 2 * distance;
    distance_travel_per_week = 5 * distance_travel_per_day;
    puts distance_travel_per_week
    test = test - 1;
end